uber/automated_emails.py

```diff
@@ -60,7 +60,7 @@ def send_all(cls, raise_errors=False):
                 attendees, groups = session.everyone()
                 approved = {ae.subject for ae in session.query(ApprovedEmail).all()}
                 models = {Attendee: attendees, Group: groups}
```

Comment:
This needs to be passed in so that we can actually perform queries in our ``lister`` function.